meta-client/src/main/java/org/apache/helix/metaclient/recipes/leaderelection/LeaderElectionListenerInterface.java

```diff
@@ -24,9 +24,16 @@
  * leader node is deleted.
  */
 public interface LeaderElectionListenerInterface {
+  enum ChangeType {
+    NEW_LEADER_ELECTED,
+    LEADER_GONE
```

Comment:
Would suggest reword it:

LEADER_ACQUIRED
LEADER_LOST

Reply:
Client can opt to listener on leader election changes and not essentially participant on leader election group. I feel like 'LEADER_ACQUIRED' is a bit confusing as it indicates the current client acquired/lost leadership